core/domain/suggestion_services.py

```diff
@@ -2965,6 +3450,221 @@ def increment_question_review_stats(
+            suggestion_models.REVIEW_OUTCOME_ACCEPTED)
+
+
+def increment_translation_reviewer_total_stats(
+    translation_reviewer_total_stat:
+        suggestion_registry.TranslationReviewerTotalContributionStats,
+    content_word_count: int,
+    last_contribution_date: datetime.datetime,
+    suggestion_is_accepted: bool,
+    edited_by_reviewer: bool
+) -> None:
+    """Updates TranslationReviewerTotalContributionStats object.
+
+    Args:
+        translation_reviewer_total_stat:
+            TranslationReviewerTotalContributionStats. The stats object to
+            update.
+        content_word_count: int. The number of words in the translation.
+        last_contribution_date: datetime.datetime. The last updated date.
+        suggestion_is_accepted: bool. A flag that indicates whether the
+            suggestion is accepted.
+        edited_by_reviewer: bool. A flag that indicates whether the suggestion
+            is edited by the reviewer.
+    """
+    translation_reviewer_total_stat.reviewed_translations_count += 1
+    if suggestion_is_accepted:
+        translation_reviewer_total_stat.accepted_translations_count += 1
+        translation_reviewer_total_stat.accepted_translation_word_count += (
+            content_word_count)
+    else:
+        translation_reviewer_total_stat.rejected_translations_count += 1
+    if suggestion_is_accepted and edited_by_reviewer:
+        (
+            translation_reviewer_total_stat
+            .accepted_translations_with_reviewer_edits_count
+        ) += 1
+    translation_reviewer_total_stat.last_contribution_date = (
+        last_contribution_date.date())
+
+
+def increment_question_submitter_total_stats_at_review(
+    question_submitter_total_stat: (
+        suggestion_registry.QuestionSubmitterTotalContributionStats),
+    suggestion_is_accepted: bool,
+    edited_by_reviewer: bool
+) -> None:
+    """Updates QuestionSubmitterTotalContributionStats object.
+
+    Args:
+        question_submitter_total_stat:
+            QuestionSubmitterTotalContributionStats. The stats object to
+            update.
+        suggestion_is_accepted: bool. A flag that indicates whether the
+            suggestion is accepted.
+        edited_by_reviewer: bool. A flag that indicates whether the suggestion
+            is edited by the reviewer.
+    """
+    # Weights for calculating performance.
+    # recent_performance = accepted cards - 2 (rejected cards) in last
+    # 100 contributions.
+    if suggestion_is_accepted:
+        question_submitter_total_stat.accepted_questions_count += 1
+        question_submitter_total_stat.overall_accuracy = round((
+            question_submitter_total_stat.accepted_questions_count
+            / question_submitter_total_stat.submitted_questions_count
+        ), 3) * 100
+
+        if (
+                len(question_submitter_total_stat
+                    .recent_review_outcomes)
+                > 99
```

Comment:
Can we have this as a constant? I see we have used the number 99 multiple times

Reply:
Updated. PTAL.